docs/intro.rst

```diff
@@ -17,9 +23,9 @@ Currently implemented functionality in this library comprises:
 * (averaged) cross spectra
 * coherence, time lags
 * cross correlation functions
-* r.m.s. spectra and lags (time vs energy, time vs frequency); UNDER DEVELOPMENT
-* covariance spectra; UNDER DEVELOPMENT
-* bispectra; UNDER DEVELOPMENT
+* RMS spectra and lags (time vs energy, time vs frequency); *under development*
```

Comment:
Now that we are at it, I would change these "under development" into "needs testing", because the methods are there, but few people used it until now.

Reply:
Consider it done (because it is).